src/main/java/frc/robot/subsystems/elevator/ElevatorSubsystem.java

```diff
@@ -182,4 +193,34 @@ public ElevatorState getTargetState() {
     public void setManualPower(double power) {
         this.manualPower = power;
     } 
+    /**
+     * Sets the motor power.
+     *
+     * @param power the power you want to set to
+     */
+
+    public void setMotorPower(double power) {
+        extensionMotor.set(power);
+    }
+    /**
+     * Returns the state of the limit switch, 0 is not triggered, 1 is triggered.
+     *
+     * @return false is not triggered, true is triggered. 
+     */
+
+    public boolean getLimitSwitch() {
+        if (zeroLimitSwitch != null) {
+            return !zeroLimitSwitch.get();
+        } else {
+            throw new Error("Can't find limit switch!");
```

Comment:
probably don't do this because it will crash the code if this is called when the limit switch is unplugged. For example, if the limit switch gets unplugged in a match, the robot code will crash when this is called. You can print something and then the best thing to would be disabling the code that uses the limit switch. (if no limit switch is detected, fallback on the current code that doesn't use it).

Reply:
Ok will work on that.